apps/challenges/models.py

```diff
@@ -315,21 +313,16 @@ def save(self, *args, **kwargs):
         return challenge_phase_instance
 
 
-def post_save_connect(field_name, sender):
-    import challenges.aws_utils as aws
+def post_save_connect(field_name, sender, callback):
     signals.post_save.connect(
         model_field_name(field_name=field_name)(
-            aws.restart_workers_signal_callback
```

Comment:
@burnerlee why are we removing restart workers callback from here and not removing the eks cluster setup callback from [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/models.py#L194)

Reply:
@Ram81 this is because the functions like `create_eks_cluster_for_challenge`are called only when post_save signal is received rather than being called during the start of evalai program. Whereas the callback function in the first case is called when the program startup which causes an error of cyclic import as all modules are initialized in the starting. So these functions import these modules inside them which actually imports the module when these functions are called. 
Whereas if I do the same for the callback function, it would import the module at the start of the execution of the program only as it is called at the start of the program which causes the same problem which is created due to importing the modules at the top of the file.